<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Life of Mine Optimizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Block Model Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>File Import</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLineEdit" name="fileCollar"/>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButtonCollarCentroids">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Select Collar Centroids CSV</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="fileBlock"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButtonBlockData">
             <property name="text">
              <string>Select Block Data CSV</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="fileRock"/>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushButtonRockInfo">
             <property name="text">
              <string>Select Rock Info CSV</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Block Size (m)</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QDoubleSpinBox" name="sizeZ"/>
           </item>
           <item row="1" column="0">
            <widget class="QDoubleSpinBox" name="sizeY"/>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QDoubleSpinBox" name="sizeX"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonInitialize">
          <property name="text">
           <string>Initialize</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Design Parameters</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>628</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
